source/win-main/MainEditor.vue

```diff
@@ -704,9 +704,16 @@ function maybeHighlightSearchResults (): void {
   }
 }
 
+//These are styles specific to dark-style.
```

Comment:
We can omit this comment (in the next line, we have `body.dark`, so it should be pretty clear that this applies to dark mode).